lobby-server/src/test/java/org/triplea/db/dao/access/log/AccessLogDaoTest.java

```diff
@@ -96,22 +101,16 @@ void searchForIp() {
 
   /** There are only 2 rows, requesting a row offset of '2' should yield no data. */
   @Test
-  @DataSet("access_log/two_rows.yml")
+  @DataSet(value = ACCESS_LOG_TABLES, useSequenceFiltering = false)
   void requestingRowsOffDataSetReturnsNothing() {
     assertThat(accessLogDao.fetchAccessLogRows(2, 1, "%", "%", "%"), hasSize(0));
   }
 
   @Test
-  @DataSet(cleanBefore = true, value = "access_log/empty_data.yml")
-  @ExpectedDataSet("access_log/insert_registered_after.yml")
-  void insertRegisteredUserAccessLog() {
-    accessLogDao.insertUserAccessRecord("registered", "127.0.0.20", "registered-system-id");
-  }
-
-  @Test
-  @DataSet(cleanBefore = true, value = "access_log/empty_data.yml")
-  @ExpectedDataSet("access_log/insert_anonymous_after.yml")
-  void insertAnonymousAccessLog() {
+  @DataSet(cleanBefore = true, value = EMPTY_ACCESS_LOG, useSequenceFiltering = false)
+  @ExpectedDataSet(value = "access_log/access_log_post_insert.yml", orderBy = "username")
+  void insertAccessLogRecords() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6092439)